src/data.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Jane Doe",
+  "nav": [
+    {
+      "name": "About",
+      "key": "#about"
+    },
+    {
+      "name": "Experience",
+      "key": "#experience"
+    },
+    {
+      "name": "Projects",
+      "key": "#projects"
+    },
+    {
+      "name": "Contact",
+      "key": "#contact"
+    }
+  ]
+}
```

Comment:
navigations should not be present in `data.json` file, create `constants.tsx` file to store these things